<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pivot Calibration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PivotCalibration.qrc">
    <normaloff>:/Images/Images/pivot.png</normaloff>:/Images/Images/pivot.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Preparation</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Select pointer tool marker id:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="idBox">
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>403</string>
                </property>
                <property name="icon">
                 <iconset resource="PivotCalibration.qrc">
                  <normaloff>:/Images/Images/403.png</normaloff>:/Images/Images/403.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>102</string>
                </property>
                <property name="icon">
                 <iconset resource="PivotCalibration.qrc">
                  <normaloff>:/Images/Images/102.png</normaloff>:/Images/Images/102.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>201</string>
                </property>
                <property name="icon">
                 <iconset resource="PivotCalibration.qrc">
                  <normaloff>:/Images/Images/201.png</normaloff>:/Images/Images/201.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>202</string>
                </property>
                <property name="icon">
                 <iconset resource="PivotCalibration.qrc">
                  <normaloff>:/Images/Images/202.png</normaloff>:/Images/Images/202.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>502</string>
                </property>
                <property name="icon">
                 <iconset resource="PivotCalibration.qrc">
                  <normaloff>:/Images/Images/502.png</normaloff>:/Images/Images/502.png</iconset>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Select number of Transformations</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QSlider" name="TransformationsSlider">
                 <property name="minimum">
                  <number>300</number>
                 </property>
                 <property name="maximum">
                  <number>3000</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="TransformationsSpinBox">
                 <property name="minimum">
                  <number>300</number>
                 </property>
                 <property name="maximum">
                  <number>3000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="CalibrationProgressBar">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Calibration Result</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="InitializeButton">
          <property name="text">
           <string>Initialize Tracker</string>
          </property>
          <property name="shortcut">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LEDlabel">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PivotCalibration.qrc">:/Images/Images/redLED.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="CheckPositionButton">
          <property name="text">
           <string>Check Marker Position</string>
          </property>
          <property name="shortcut">
           <string>C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="Result">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="CalibrateButton">
          <property name="text">
           <string>Calibrate</string>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="QuitPushButton">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PivotCalibration.qrc">
              <normaloff>:/Images/Images/ButtonQuit.png</normaloff>:/Images/Images/ButtonQuit.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Q</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="logoLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="PivotCalibration.qrc">:/Images/Images/SZI_V_cmyk.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="helpButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PivotCalibration.qrc">
              <normaloff>:/Images/Images/help.png</normaloff>:/Images/Images/help.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="shortcut">
             <string>H</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="PivotCalibration.qrc"/>
 </resources>
 <connections/>
</ui>
